srtcore/buffer.h

```diff
@@ -429,12 +429,17 @@ class CRcvBuffer
     /// if none
     /// @param [out] w_passack   true if 1st ready packet is not yet acknowleged (allowed to be delivered to the app)
     /// @param [out] w_skipseqno SRT_SEQNO_NONE or seq number of 1st unacknowledged pkt ready to play preceeded by
+    /// @param base_seq          SRT_SEQNO_NONE or desired, ignore seq smaller than base if exist RTP larger than base
```

Comment:
RTP here is a bit ambiguous with the RTP protocol.
Did you mean "ready to play"?

Reply:
yes, since I saw `RTP` also used in the comment of `getRcvFirstMsg()`
`w_skipseqno == SRT_SEQNO_NONE: no packets to skip towards the first RTP`
so I thought it's a common abbr in SRT, LOL, I can fix it :)